Gemfile

```diff
@@ -1,3 +1,46 @@
 source 'https://rubygems.org'
 
 gemspec
+
+gem 'jruby-openssl', '~> 0.9.10', platform: :jruby
+
+gem 'aruba', '< 0.7', require: false, group: :development
+
+gem 'activesupport', '~> 3.2' if RUBY_VERSION < '2.0'
+gem 'mime-types', '< 2.99' if RUBY_VERSION < '2.0'
+
+if RUBY_VERSION < '2.0'
+  if ENV['AS_VERSION']
+    gem 'activesupport', ENV['AS_VERSION']
+  else
+    # NOTE: keep testing against AS 3.2.x by default
+    # :require needed as otherwise some specs fail :
+    # undefined method `deep_merge' for {:from=>"foo", :timeout=>3000, :headers=>{:x_foo=>"bar"}}:Hash
+    gem 'activesupport', '~> 3.2', require: ['active_support', 'active_support/core_ext/hash']
+  end
+  gem 'mime-types', '< 2.99'
+end
+
+if pb_version = ENV['PB_VERSION']
```

Comment:
yy that was the intention ... update [PB_VERSION](https://github.com/kares/adhearsion/blob/4927ad41eacd4199259c03da1ca305a76540f75d/.travis.yml#L23-L32) to `dialogtech/support/2.7.5` once merged